scripts/fbi/hate_crime/download_publication_data.py

```diff
@@ -0,0 +1,363 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A script to scrape and download FBI Hate Crime table publications data."""
+import requests
+import os
+from bs4 import BeautifulSoup
```

Comment:
please group imports into the following blocks, each sorted alphabetically if there are no interdependancies:


import abc
import def

from ghi import myfunc
from lmn import pqr

Reply:
fixed.